pkg/generated/protobuf/fulcio.pb.go

```diff
@@ -16,7 +16,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.20.3
+// 	protoc        v3.21.11
```

Comment:
Ah, this is gonna be an issue. My recommendation is to either update https://github.com/sigstore/fulcio/blob/d1fa542e81caff3583337a990898bcd701f0aa7f/.github/workflows/main.yml#L41, or install 3.20.3. We need to use a containerized protoc, we have an issue tracking this

Reply:
My `protoc` is from `brew`, so downgrading might be a little painful 😅 -- I'll bump the action if that's alright.

Reply:
https://github.com/sigstore/fulcio/pull/926/commits/05bc547af17d3e69919a7702b5a1a84acdff35a7

Reply:
Hm, it seems to think that version doesn't exist.

Reply:
Yeah...looks like manifestation of this issue: https://github.com/arduino/setup-protoc/issues/33

Reply:
(It seems like protobuf changed its versioning to chop the `3.` prefix off, and this action doesn't know about that.)

Reply:
I opened https://github.com/arduino/setup-protoc/pull/58 with a fix to that action, and I've temporarily updated this PR to pin my `trail-of-forks` fork instead. But I assume we'll want this to be upstreamed before merging 🙂